controllers/homework.js

```diff
@@ -162,169 +156,162 @@ router.post('/submit', getCreateHandler('submissions'));
 router.post('/comment', getCreateHandler('comments'));
 router.delete('/comment/:id', getDeleteHandlerR('comments'));
 
+
+
+const addLeadingZero = function(s) {
+    return (s < 10) ? "0" + s : s;
+}
+const splitDate = function(date){
+    var realDate = new Date(new Date(date).getTime() + (new Date(date).getTimezoneOffset() * 60000));
+    var DateF = addLeadingZero(realDate.getDate()) + "." + addLeadingZero(realDate.getMonth() + 1) + "." + realDate.getFullYear();
+    var TimeF = addLeadingZero(realDate.getHours()) + ":" + addLeadingZero(realDate.getMinutes());
+    return {"timestamp":realDate,"date":DateF,"time":TimeF};
+}
+const formatDateDiff = function(datediff){
+    var Days = Math.floor(datediff / 86400000);
+    var Hours = Math.floor((datediff % 86400000) / 3600000);
+    var Minutes = Math.floor(((datediff % 86400000) % 3600000) / 60000);
+    return {"Days":Days,"Hours":Hours,"Minutes":Minutes};
+}
+const formatremaining = function(remaining){
```

Comment:
Like I mentioned before in #dev channel - please use instead of a custom solution: 
```js
moment(dueDate).toNow();
```